imports/ui/side-menu/side-menu.jsx

```diff
@@ -66,23 +82,11 @@ class SideMenu extends Component {
         })}
         <Divider />
         {this.linkDrawerItem({
-          href: 'https://unee-t.com/contact-support/',
+          href: 'mailto:support@unee-t.com?subject=' + window.location.href + '&body=' + encodeURIComponent(supportEmailBody),
           iconName: 'live_help',
           text: 'Support',
           isExternal: true
         })}
-        {this.linkDrawerItem({
```

Comment:
Why did you remove these 2 links? I thought we wanted them in

Reply:
Forum is gone. FAQ has limited utility too. I can add them back @franck-boullier